test_app/lib/screens/participant_screen/participant_screen.dart

```diff
@@ -12,7 +12,11 @@ import '/widgets/dolby_title.dart';
 import '/widgets/modal_bottom_sheet.dart';
 
 class ParticipantScreen extends StatefulWidget {
-  const ParticipantScreen({Key? key}) : super(key: key);
+  final bool isSpatialAudio;
+  final bool isDolbyVoice;
+  const ParticipantScreen(
+      {Key? key, required this.isSpatialAudio, required this.isDolbyVoice})
```

Comment:
Why do you need both `this.isSpatialAudio` and `this.isDolbyVoice`? Why do we allow spatial audio to be true if we have a non dolby voice conference.

Reply:
Aga is working on that issue. After my PR it will be refactored

Reply:
My question is about why you need both variables?

Reply:
Because Spatial Audio doesn't work without Dolby Voice.

Reply:
We have agreed that we are going to prohibit entering the participant screen when
* isSpatialAudio = ture, and
* isDolbyVoice = false. 
What do you think of my question in the light of this information?

Reply:
We don't need both this.isSpatialAudio and this.isDolbyVoice and we don't allow spatial audio to be true if we have a non dolby voice conference.

Reply:
Ok, thank you for this. Just another question that popped into my head: is there any way of finding out whether a conference uses spatial audio or not from the conference object?

Reply:
Our conference object doesn't contain informations about spatial audio.

Reply:
Then it is fine.